server/db/schema.js

```diff
@@ -0,0 +1,19 @@
+var mongoose = require('mongoose');
+
+mongoose.connect('mongodb://localhost/cushion');
+
+var TechieSchema = new mongoose.Schema({
+            userName: {type: String, unique: true},
```

Comment:
Let's correct the tabbing here to be in line with the style guide! (I think it's two spaces, you shouldn't have to use the tab key :)